apps/src/code-studio/initApp/loadApp.js

```diff
@@ -24,11 +24,14 @@ import {
 } from '@cdo/apps/code-studio/levels/codeStudioLevels';
 import queryString from 'query-string';
 import { dataURIToFramedBlob } from '@cdo/apps/imageUtils';
+import i18n from '@cdo/locale';
+import experiments from '@cdo/apps/util/experiments';
 
 // Max milliseconds to wait for last attempt data from the server
 var LAST_ATTEMPT_TIMEOUT = 5000;
 
 const SHARE_IMAGE_NAME = '_share_image.png';
+const POINTS_KEY = 'tempPoints';
```

Comment:
Prefer sharing this const with [feedback.js](https://github.com/code-dot-org/code-dot-org/blob/1ba9962e332a69b14f6996078ac8cc44df58bda5/apps/src/feedback.js#L46), but for the experiment this is fine.